test/unit/data_hygiene/statistics_announcement_unpublisher_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+require 'gds_api/test_helpers/router'
+
+module DataHygiene
+  class StatisticsAnnouncementUnpublisherTest < ActiveSupport::TestCase
+    include GdsApi::TestHelpers::Router
+
+    setup do
+      @fake_logger = mock()
+      @fake_logger.stubs(:info)
+
+      @announcement = create(:statistics_announcement)
+    end
+
+    test "when given a non-existent slug" do
```

Comment:
Ack, yes, lost that in some refactoring which we then backed out. I'll fix the test descriptions